luminoso_api/json_stream.py

```diff
@@ -237,10 +241,12 @@ def _read_csv(reader, header, encode_fn):
         if row_dict.get('title') == '':
             del row_dict['title']
         if 'date' in row_dict:
+            # We handle dates further in stream_json_lines, but if date_format
+            # is not specified and the dates are not in epoch time, we don't
```

Comment:
I'm not sure I understand this comment.  How can `date_format` not be specified, as far as the upload script is concerned?  Doesn't it have a default value?